regression/rt_gk_sheath_1x2v_p1.c

```diff
@@ -139,20 +141,19 @@ create_ctx(void)
   int Nmu = 4; // Cell count (velocity space: magnetic moment direction).
   double Lz = 4.0; // Domain size (configuration space: z-direction).
   double vpar_max_elc = 4.0 * vte; // Domain boundary (electron velocity space: parallel velocity direction).
-  double mu_max_elc = (3.0 / 2.0) * 0.5 * mass_elc * pow(4.0 * vte,2) / (2.0 * B0); // Domain boundary (electron velocity space: magnetic moment direction).
+  double mu_max_elc = (3.0 / 2.0) * 0.5 * mass_elc * pow(4.0 * vte, 2.0) / (2.0 * B0); // Domain boundary (electron velocity space: magnetic moment direction).
   double vpar_max_ion = 4.0 * vti; // Domain boundary (ion velocity space: parallel velocity direction).
-  double mu_max_ion = (3.0 / 2.0) * 0.5 * mass_ion * pow(4.0 * vti,2) / (2.0 * B0); // Domain boundary (ion velocity space: magnetic moment direction).
+  double mu_max_ion = (3.0 / 2.0) * 0.5 * mass_ion * pow(4.0 * vti, 2.0) / (2.0 * B0); // Domain boundary (ion velocity space: magnetic moment direction).
+  int poly_order = 1; // Polynomial order.
+  double cfl_frac = 1.0; // CFL coefficient.
 
   double t_end = 6.0e-6; // Final simulation time.
   int num_frames = 1; // Number of output frames.
-  int int_diag_calc_num = num_frames*100;
```

Comment:
why are you getting rid of this?

Reply:
See above for my comment regarding integrated diagnostics. Either we calculate these things uniformly across every test, or we need to design an appropriate Lua interface so that these kinds of options can be passed in. We can't have random bespoke code in a C test if the whole idea is to facilitate comparisons against Lua tests that do not contain such bespoke code.

Reply:
this isn't bespoke code, it's important production-worthy code. If by bespoke you mean single-purpose, then the IWL feature in GK is bespoke, but that is the avenue for a whole research program (with clopen sims)!
Back to this point: if your motivation is to standardize across other models or across all GK tests, then I think the standard should be to allow for different diag rate for integrated, conf and phase space diagnostics, as I think is the case in rt_gk_sheath_3x2v_p1. That is truly the most valuable way to portray and use the GK solver